gpt_index/indices/query/base.py

```diff
@@ -242,6 +242,14 @@ def _query(self, query_bundle: QueryBundle) -> Response:
         node_texts = []
         for node, similarity in tuples:
             text, response = self._get_text_from_node(query_bundle, node)
+            if isinstance(node.extra_info, dict):
```

Comment:
hm i have a slightly different idea in mind where we can have the knowledge graph query subclass a function like `_get_extra_info_for_response`, to put all the KG triplets in the extra_info in the Response object. going to play around with it on this PR and keep you updated!